assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/rh-che.properties

```diff
@@ -41,3 +41,7 @@ che.fabric8.end2end.protect.secret_key=NULL
 # End2End Flow bot protection: option to enable the use of the client IP
 # in the captcha server-side verification 
 che.fabric8.end2end.protect.verify_with_ip=false
+
+# Full identifier of the default editor whose CDN resources
+# should be prefetched during the dashboard client-side load 
+che.fabric8.cdn.prefetch.editor=NULL
```

Comment:
Could we include an example value here? I don't know offhand what the full identifier would look like.